formapp/app.py

```diff
@@ -772,6 +769,25 @@ def endpoint_addfighteropponent():
     name = request.args.get('name')
     sD.fighter['match' + code]['opponent'] = name #Set "opponent" of object to arg "name"
 
+'''
+/addfile
+'''
+@app.route('/addfile', methods = ['POST', 'GET'])
+def endpoint_addfile():
+    if not request.remote_addr in sD.studentDict:
+        return redirect('/login?forward=' + request.path)
+    else:
+        if request.method == 'POST':
+            title = request.form['title']
+            file = request.form['file']
+            list = request.form['list']
+            print("Title: " + title)
+            print("Filename: " + file)
+            print("List: " + list)
+            return render_template("message.html", forward=request.path, message = 'File submitted to teacher.')
+        else:
+            return render_template('addfile.html')
+
```

Comment:
This is cool, but you should only work on features in a branch that the branch was intended for. I didn't know I was supposed to check for this, and I might have missed it and added features that weren't intended.

You should also focus on only features in Open Issues as well. If you have ideas for new features, please add them as an Issue card so the team can analyze their importance/priority.

If you are PR'ing multiple new features, please list them all in the PR description.

Reply:
I added this as issue #129.